src/BuildScriptGenerator/Node/NodePlatformInstaller.cs

```diff
@@ -17,12 +17,12 @@ public class NodePlatformInstaller : PlatformInstallerBase
         {
         }
 
-        public override string GetInstallerScriptSnippet(string version)
+        public virtual string GetInstallerScriptSnippet(string version)
```

Comment:
`virtual` here allows unit tests to override and return mock script.

Reply:
This is a great idea. Maybe change it inside Dotnetcore and Php platform as well?

Reply:
Sure 👍